pages/management/applications.md

```diff
@@ -66,6 +66,40 @@ When you create a view, you are saving a specific set of filters that you may wa
 
 You will be asked to choose a name for the saved view. This view can then be selected at any time by clicking the *Views* button above the filter box.
 
+## Device tags
```

Comment:
we could explain that tags are created and stored as key:value pairs

Reply:
and perhaps that the name/key is unique and that the value is optional.

Also, the [spec describes](https://github.com/resin-io/hq/pull/281/files#diff-855f6128af9496112f322bd42abfe5ddR29) that we should promote using the `reverse domain notation` for tag keys/names. Even though not mandatory or enforced with any way, it can be really useful since it leads to structured tag names. I wonder whether this fits in here. Maybe not.

Reply:
@thgreasi I think that will be most useful to have in the API documentation around tags, but I'm not sure. Could you provide an example where using that structure would be helpful?

Reply:
You mean the `reverse domain notation`?
For example you can have several `stats` and `status` related tags and keep them organized:
* stats.boot.lastbootdate
* stats.boot.lastbootduration
* status.disk.sizeavailable
* status.myappspecificstate (eg recording video)

On second thought, maybe it's too much to have it in this page. I'm fine with this as is.